controller/src/main/java/io/pravega/controller/store/stream/StoreException.java

```diff
@@ -23,33 +24,38 @@
      * Enum to describe the type of exception.
      */
     public enum Type {
-        NODE_EXISTS,
-        NODE_NOT_FOUND,
-        NODE_NOT_EMPTY,
+        DATA_EXISTS,
+        DATA_NOT_FOUND,
+        DATA_NOT_EMPTY,
```

Comment:
Adding @fpj comment from a different PR here:
"I only have one comment about node not empty referring to a parent that still has children. I'm not sure there is anything we can do given that it is an exception type that exists already, but if we can give a more descriptive name, then better"